strapiExperiment/docker-compose.yml

```diff
@@ -0,0 +1,59 @@
+version: "3"
+
+services:
+  reactexample:
+    container_name: frontend
+    build: 
+      context: ./testreact
+      dockerfile: Dockerfile
+    networks:
+      - strapi-app-network
+    volumes:
+      - '.:/testreact/app'
+      - '/testreact/app/node_modules'
```

Comment:
this can cause issues on macs. The node_modules volume can contain different binaries between mac os and linux distributions